content/docs/self-hosted-runners.md

```diff
@@ -207,25 +245,52 @@ Click below to see credentials needed for supported cloud service providers.
 
 </details>
 
-## Using on-premise machines as self-hosted runners
+<details>
+
+#### GCP
+
+Either one of:
+
+- `GOOGLE_APPLICATION_CREDENTIALS_DATA`: the **contents** of a service account
+  JSON file, or
+- `GOOGLE_APPLICATION_CREDENTIALS`: the **path** to the JSON file.
```

Comment:
```suggestion
- `GOOGLE_APPLICATION_CREDENTIALS_DATA` with the **contents** of a service account
  JSON file, or
- `GOOGLE_APPLICATION_CREDENTIALS` with the **path** to the JSON file.
```

The OCD I Don't Have ™ doesn't like the style of punctuation symbols after a `<code>` inline block. This is an optional suggestion that doesn't necessarily provide any improvement for other readers.